src/upcoming-components/Accordion/Accordion.test.tsx

```diff
@@ -0,0 +1,8 @@
+// import { generateSnapshots } from '@chanzuckerberg/story-utils';
+// import * as stories from './Accordion.stories';
+
+describe('<Accordion />', () => {
+  // TODO: uncomment this and delete the skipped test when moving this component into the main src/components directory
+  // generateSnapshots(stories);
+  test.skip('Workaround', () => undefined);
```

Comment:
Any reason to not snapshot it yet? Is it incomplete or something?

Reply:
We talked about this in a meeting but I'll post it here for posterity – the components in `upcoming-components` are very much in a WIP state and some may never be used, so it feels like bloat to me to snapshot all of them and have to accept the changes when components they depend on change. But since we will have to remember to uncomment-out these test files when we start serious work on any of these components (and we often forget stuff like that), what we're going to do instead is exclude this whole directory from the test runners. That way we can have the fully functional test files, but they just won't run until the component is moved into the active `components` directory.